Libraries/Text/Text.js

```diff
@@ -90,6 +91,10 @@ const Text = React.createClass({
      */
     onLongPress: React.PropTypes.func,
     /**
+     * Lets the user select text, to use the native copy and paste functionality.
```

Comment:
I think `selectable` is correct here, we will want to support this on iOS as well. The platform suffix makes more sense when the prop is platform specific and cannot/will not be supported on the other platform.